spec/factories/users.rb

```diff
@@ -165,8 +165,8 @@
       loa do
         { current: LOA::THREE, highest: LOA::THREE }
       end
-
-      idme_uuid { '6767671' }
+      vet360_id { '6767671' }
```

Comment:
I think instead of having this trait :error in the user factory, it probably just makes sense to define the attribute combination to cause the error for your specific service in the spec file itself. This isn't generally a user error in any other situation so it's difficult for any other spec to leverage this, which is what would be the reason to put a trait in the factory

Reply:
Idme uuid also has to be null. The `:user, :error` factory was used to validate invalid user requests.

Reply:
But the user model is generally fine without an `idme_uuid` or a `vet360_id`, so this doesn't actually describe an error state for the `User` model, it only describes an error when interacting with this specific service

Reply:
Yes, I added a trait to the user factory for the contact service. I'll just remove the trait.

Reply:
I prefer to use factory settings instead of hard coding values into tests. It's easier to see which VCR cassette is used with a certain factory.

Reply:
@bosawt Done!